### PR TITLE
Update readme to clarify current state of schema validation

### LINKED ISSUE #41: Update README and description to clarify syntax only validation
# Description

Feedback from some developers was that schema validation was expected based on the name of the tool. The README and Github project description should be updated to clarify that only syntax validation is provided (until #40 is worked)

### PR DESCRIPTION
**Pull request by luancgs**
This PR addresses issue #41, to clarify state of schema validation while #40 is not resolved.

closes #41

### CODE REVIEW

**Comment by luancgs** [comment]
@kehoecj Let me know if this works for you or if there is something that needs change

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ go install github.com/Boeing/config-file-validator/cmd/validator
 The config-file-validator is built as a statically linked binary which can be downloaded and executed on your target system. Binaries are available for Linux, Windows, and MacOS. Navigate to the [releases](https://github.com/Boeing/config-file-validator/releases) page to download the latest version. Once the binary has been downloaded it needs to be installed by moving the downloaded file to a location on your operating system's PATH.
 
 ## Using
+
+> [!IMPORTANT]
```
Comment (kehoecj) [inline README.md:56]: I feel like the `About` section is more appropriate for this type of clarification. Also, I'm not sure this needs an `[!IMPORTANT]` highlight as this was intedented to just be a clarification - `[!NOTE]` is more appropriate.

**Review by kehoecj — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (luancgs) [reply README.md:56]: Sure, let me fix it

File: README.md [hunk 2]
```diff
@@ -24,6 +24,9 @@
 ## About
 How many deployments have you done that needed to be rolled back due to a missing character in a configuration file in your repo? If you're like most teams that number is greater than zero. The config file validator was created to solve this problem by searching through your project and validating the syntax of all configuration files. 
 
+> [!NOTE]
+> For now, the config-file-validator is a tool that specializes in syntax validation. It does **not** perform schema validation.
```
Comment (kehoecj) [inline README.md:28]:
```suggestion
> The config-file-validator only performs syntax validation. We are working toward adding support for schema validation in future releases
```

**Pull request merged**